docs/Tuners.md

```diff
@@ -0,0 +1,265 @@
+|**Batch tuner**|Batch tuner allows users to simply provide several configurations (i.e., choices of hyper-parameters) for their trial code. After finishing all the configurations, the experiment is done. Batch tuner only supports the type choice in search space spec.|If the configurations you want to try have been decided, you can list them in searchspace file (using choice) and run them using batch tuner.|Usage: [BatchTunerUsage][10]|
+|**Grid Search**|Grid Search performs an exhaustive searching through a manually specified subset of the hyperparameter space defined in the searchspace file. Note that the only acceptable types of search space are choice, quniform, qloguniform. The number q in quniform and qloguniform has special meaning (different from the spec in search space spec). It means the number of values that will be sampled evenly from the range low and high.|It is suggested when search space is small, it is feasible to exhaustively sweeping the whole search space.|Usage: [GridSearchUsage][11]|
+|**Hyperband**|Hyperband tries to use limited resource to explore as many configurations as possible, and finds out the promising ones to get the final result. The basic idea is generating many configurations and to run them for small number of STEPs to find out promising one, then further training those promising ones to select several more promising one.|It is suggested when you have limited computation resource but have relatively large search space. It performs good in the scenario that intermediate result (e.g., accuracy) can reflect good or bad of final result (e.g., accuracy) to some extent.|Usage: [HyperbandUsage][12] Paper: [Hyperband: A Novel Bandit-Based Approach to Hyperparameter Optimization][13]|
+|**Network Morphism**|Network Morphism provides functions to automatically search for architecture of deep learning models. Every child network inherits the knowledge from its parent network and morphs into diverse types of networks, including changes of depth, width and skip-connection. Next, it estimates the value of child network using the history architecture and metric pairs. Then it selects the most promising one to train.|It is suggested that you want to apply deep learning methods to your task (your own dataset) but you have no idea of how to choose or design a network. You modify the example to fit your own dataset and your own data augmentation method. Also you can change the batch size, learning rate or optimizer. It is feasible for different tasks to find a good network architecture. Now this tuner only supports the cv domain.|Usage: [NetworkMorphismUsage][14] Paper: [Auto-Keras: Efficient Neural Architecture Search with Network Morphism][15]|
+
```

Comment:
Note: could add metis tuner to here later

Reply:
Add already~